airframe-http/.jvm/src/main/scala/wvlet/airframe/http/router/HttpResponseCodec.scala

```diff
@@ -23,8 +23,8 @@ class HttpResponseCodec[Resp: HttpResponseAdapter] extends MessageCodec[HttpResp
   override def pack(p: Packer, v: HttpResponse[_]): Unit = {
     v.contentType match {
       case Some("application/x-msgpack") =>
+        // Raw msgpack response
         val b = v.contentBytes
-        p.packArrayHeader(b.length)
```

Comment:
No need to call `packBinaryHeader`?

Reply:
In this case, the server generates a raw message pack value (It can be MapValue, StringValue, etc.), so returning the byte array (= msgpack) as is.